differ/storage.go

```diff
@@ -584,7 +625,7 @@ func (storage DBStorage) CleanupForOrganization(orgID types.OrgID, maxAge string
 //
 // The method return number of deleted records.
 func (storage DBStorage) CleanupNewReportsForOrganization(orgID types.OrgID, maxAge string) (int, error) {
```

Comment:
We don't have any way to call this one, right?

Reply:
nope AFAIK, it was prepared for weekly report functionality, where this cleanup was planned

Reply:
Let's keep them until we have some input on that feature then =)